src/components/Forms/PostForm/PostForm.js

```diff
@@ -37,8 +39,8 @@ export const PostForm = ({
 
   const onSubmit = (e) => {
     e.preventDefault();
-    if (!formData.label.trim() || !formData.title.trim() || !formData.content.trim()) {
-      setForm({ label: 'general', title: '', content: '' });
+    if (!formData.title.trim() || !formData.content.trim()) {
+      setForm({ label: '', title: '', content: '' });
```

Comment:
If one of the necessary field is empty, you should not clear all input. Otherwise

Reply:
I agree